superset/views/core.py

```diff
@@ -2169,6 +2169,8 @@ def dashboard(self, dashboard_id):
             else:
                 dashboard_view = 'v1'
                 prompt_v2_conversion = not force_v1
+                if force_v2_edit:
```

Comment:
is this needed for removing the `save as` dropdown option in a `v1` app? Does this still allow you to click edit when viewing a `v1` dashboard to go to the `v2` page?